doc/releases/versioning.adoc

```diff
@@ -1,23 +1,23 @@
-= Versioning and compatibility
+= Versioning and Compatibility
```

Comment:
@pasched Why was this rolled back? This aren't release notes of a older release.

Reply:
Fixed it myself.